utilities/pipelines/resourceRemoval/helper/Remove-Deployment.ps1

```diff
@@ -117,12 +119,16 @@ function Remove-Deployment {
         # ===============================
         $dependencyResourceNames = Get-DependencyResourceNameList
 
-        if ($resourcesToIgnore = $resourcesToRemove | Where-Object { (Split-Path $_.resourceId -Leaf) -in $dependencyResourceNames }) {
+        $resourceIdsToIgnore = @(
```

Comment:
I'd need to go through the change a bit more in depth, but for now I'd suggest to add a paragraph in the [resource removal](https://github.com/Azure/ResourceModules/wiki/The%20CI%20environment%20-%20Deployment%20validation#resource-removal) Wiki section, mentioning exceptions. Currently it's only the `NetworkWatcherRG` but I'm planning to also update that section with some of the resources part of the azure security center module

Reply:
Absolutly agree, great idea. I think this section (both in the Wiki & code) will anyways be updated to quite an extend once all dependencies converted and can be removed

Reply:
Updated :)